### DISCUSSION TITLE
Inference Rule

### DISCUSSION THREAD

**Discussion by HLe4s-kaist** [prosyslab-classroom/cs524-program-analysis#431] [Art competition]
![Inference Rule](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/161300429/3159cd9e-ca7f-4dfa-a8e4-f1e244e24cf9)
(English follows)
## Name
강찬우

## Tool You Used
[Adobe Fire Fly](https://firefly.adobe.com/upload/inpaint), [bing image creator](https://www.bing.com/images/create)

## Short Description
여러 개의 케이크가 하나의 케이크를 받치고 있다. 그런 식으로 만들어진 케이크 탑이 있다 !

## Long Description (up to 3 sentences)
Inference Rule을 생각해보면, 하나의 사실을 이끌어내기 위해서, 몇 가지의 사실을 끌어오는 것이 반복되는 그림이 생각났습니다. 그래서 비슷한 것들이 여러 개 모여서 하나를 떠 받치고, 그 중 하나라도 빠지면 모두가 무너지는 그림을 생각하며 그려봤습니다. 이 그림에서 재미있는 부분은 케이크를 단 하나의 케이크가 받치는 경우도 있고, 여러 케이크가 받치는 경우도 있다는 것입니다. 

[Eng]
## Name
ChanU Kang

## Tool You Used
[Adobe Fire Fly](https://firefly.adobe.com/upload/inpaint), [Bing Image Creator](https://www.bing.com/images/create)

## Short Description
Multiple cakes are supporting a single cake, forming a cake tower!

## Long Description (up to 3 sentences)
When thinking about the Inference Rule, I envisioned a repetitive process of drawing upon several facts to derive a single fact. This led me to create an illustration where multiple similar objects support one, forming a structure that would collapse if any one of them were removed. The interesting part of this drawing is that sometimes a single cake supports another cake, while in other instances, multiple cakes do the supporting.

**Comment by KihongHeo** [comment]
Good. BTW, why cake? (because here is Daejeon?)

**Comment by HLe4s-kaist** [comment]
[Kor]
무언가가 여러 개 쌓여있는 모습을 상상하니, 케이크가 먼저 생각나서 그랬습니다.
'그 대전 빵집'은 생각 못해봤는데, 말씀해주셔서 생각이 나네요. 조만간 한 번 들러서 케이크 사먹어야겠습니다. 
[Eng]
Thinking of something stacked up in layers made me think of cake first. I hadn't thought about "That Daejeon Bakery," but your mention of it reminded me. I should stop by soon to buy a cake.
